template/{{cookiecutter.repository_name}}/requirements.txt

```diff
@@ -8,6 +8,7 @@ cachetools==4.1.0
 certifi==2020.4.5.1
 chardet==3.0.4
 click==7.1.2
+commitizen==2.1.0
```

Comment:
should this be a requirement?

Reply:
It's a requirement in the template, not for ml-workflow.

Reply:
yes, still the same question
there is a difference between that we enforce it on our stuff or that we enforce it on anyone creating a template